<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="background.qrc">:/new/prefix1/graph.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>显示路由表：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboTable">
       <item>
        <property name="text">
         <string>Router0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnShow1">
       <property name="text">
        <string>显示</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据流预测：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboFlow">
       <item>
        <property name="text">
         <string>Router0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnShow2">
       <property name="text">
        <string>显示</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCompute">
       <property name="text">
        <string>开始仿真</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConver">
       <property name="text">
        <string>单步收敛</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDynamicRouting">
    <property name="title">
     <string>DynamicRouting</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuDynamicRouting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>更新数据流</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
